xla/service/gpu/autotuning/gemm_fusion_autotuner.h

```diff
@@ -183,11 +185,21 @@ class GemmFusionAutotunerImpl {
                               const ScopedShapedBuffer& buffer,
                               AutotuneResult& res);
 
-  se::CudaComputeCapability GetComputeCapability() const {
-    return std::get<se::CudaComputeCapability>(
-        config_.GetGpuComputeCapability());
+  se::GpuComputeCapability GetComputeCapability() const {
+    return config_.GetGpuComputeCapability();
+  }
+
+  bool isRocm() const {
+    return std::holds_alternative<se::RocmComputeCapability>(
+          GetComputeCapability()) ? true : false;
```

Comment:
Please remove `? true : false`, the result of `holds_alternative` is already what you want.